### ISSUE TITLE
Any plan to add VS2022 Support

### ISSUE THREAD

**Issue by peopleworks** [foryoung365/vs-extension-llmcopilot#9]
Hello @foryoung365 

Any plan to add VS2022 Support, I think is the most used version.

Thank you

**Comment by foryoung365** [comment]
@peopleworks Yes, you are right. So, you can just open vs2022's extension manager or marketplace and search for `LLMCopilot`, it will be there.

**Comment by peopleworks** [comment]
Excellent !! Thank you.
